packages/server-ct/src/routes-ct.ts

```diff
@@ -86,6 +86,19 @@ export const createRoutes = ({
     })
   })
 
+  // enables runner-ct to make a dynamic import
+  app.get(`${config.clientRoute}ctChunk-*`, (req, res) => {
+    debug('Serving Cypress front-end chunk by requested URL:', req.url)
+
+    serveChunk(req, res, { config })
+  })
+
+  app.get(`${config.clientRoute}vendors~ctChunk-*`, (req, res) => {
+    debug('Serving Cypress front-end vendor chunk by requested URL:', req.url)
+
+    serveChunk(req, res, { config })
+  })
```

Comment:
This needs only for **us** to make possible to use dynamic export. 
Restricting it to a specific namespace (`ctChunk-`) in order to avoid any kind of proxying issues.

Reply:
You probably should add a comment about this in the code ;)

Reply:
`  // enables runner-ct to make a dynamic import` is it enough? line 89